shared_context/active_filters_shared.rb

```diff
@@ -0,0 +1,102 @@
+require "rspec"
+require "watir"
+require "helper"
+
+shared_context "Filter Categories" do |sweaters, shoes, shirts, trousers, hats|
```

Comment:
same here, use single quotes 'Filter Categories'

Reply:
Try to use hash array as a parameter instead of set of provided parameters (sweaters, shoes, shirts, trousers, hats).   Because in this case we have 5 parameters and by writing code this way it will be easier to search values location (e.g.). More: http://ruby-for-beginners.rubymonstas.org/built_in_classes/hashes.html

For example:
```
shared_context 'Filter Categories' do |filter_categories_data|

sweaters = filter_categories_data['sweaters']
shoes = filter_categories_data['shoes']
shirts = filter_categories_data['shirts']
trousers = filter_categories_data['trousers']
hats = filter_categories_data['hats']
```

so in regression_filter.rb  it will be instead of:
```
sweaters = "Clothes/Sweaters x"
   shoes = "Clothes/Shoes x"
   shirts = "Clothes/Shirts x"
   trousers = "Clothes/Trousers x"
   hats = "Clothes/Hats x"

   include_context 'Filter Categories', sweaters, shoes, shirts, trousers, hats
```

this:

 ```
filter_categories_data = {
                          'sweaters' => 'Clothes/Sweaters x',
                          'shoes' => 'Clothes/Shoes',
                          'shirts' => 'Clothes/Shirts x',
                          'trousers' => 'Clothes/Trousers x'
                          'hats' => 'Clothes/Hats x'
                          }

include_context 'Filter Categories', filter_categories_data
```